src/styles/components/_article-preview.scss

```diff
@@ -1,35 +1,77 @@
 .article-preview {
   min-width: 220px;
+  max-width: $column-3-1;
```

Comment:
Sorry I'm a little unclear about these max-width properties. Do you need them at all with the flex option? Couldn't you just remove these and jsut use flex to set the width relative to the parent?